bin/github_download_release.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+#
+# gh-dl-release! It works!
+#
+# Source https://gist.github.com/maxim/6e15aa45ba010ab030c4
+#
+# This script downloads an asset from latest or specific Github release of a
+# private repo. Feel free to extract more of the variables into command line
+# parameters.
+#
+# PREREQUISITES
+#
+# curl, wget, jq
```

Comment:
I decided to leave this code in sh script and call it in github:download-release
Because this is complex script and I do not want to put it in Makefile